packages/terra-clinical-result/src/proptypes/observationPropTypes.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types';
+import interpretationPropType from './interpretationPropTypes';
+import statusPropType from './statusPropTypes';
```

Comment:
This proptype is only used here from what I can tell. Can we remove the reusable proptypes and move it here?

Reply:
It's now used in multiple places.